### ISSUE TITLE
508-defect-3 [COGNITION]: Start your application button should be written as a link and styled as an action link

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24107]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

- As a user, I expect links to be styled as links so that I know I can open them in a new tab.
- As a voice AT user, I expect links to be styled as links so that I can provide an accurate command based on what I see.
- As a screen reader user with low vision, I expect the screen reader to announce what I see on the page.
- As a user, I expect links to be styled consistently as links so that I do not confuse them with buttons.

## Details

The "Go to eBenefits" link within the eligibility page is written and styled as a `button` which is materially dishonest because it redirects the user to a new page. This causes several issues:
- It's materially dishonest and damages habituation on VA.gov 
- Screen reader users will expect a change on the page, not a redirect to a new page
- Users won't be able to take advantage of "open in a new tab" which comes with `a` functionality

This is a sitewide long-tail effort. [To better track button/link inconsistencies, please view this Mural.](https://app.mural.co/invitation/mural/vsa8243/1610052905994?sender=u2134c22982ad9c7b41798011&key=87284ca2-fd99-431d-b086-5890abf480ac).

## Acceptance Criteria

- [ ] Start your application button is written as a link
- [ ] Start your button is styled as an action link

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Solution 

Use the new action link component in the VA design system

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [WCAG Success Criterion 3.2.4: Consistent Identification](https://www.w3.org/WAI/WCAG21/Understanding/consistent-identification.html)

## Screenshots or Trace Logs

<img width="1547" alt="Screen Shot 2021-05-03 at 11 45 24 AM" src="https://user-images.githubusercontent.com/14154792/116899540-a9e36c80-ac05-11eb-96a9-e7fd03ce3ebf.png">

**Comment by joshkimux** [comment]
Note-- this may be out of scope for this effort given that the review is for the new sign functionality in particular. If so, please rout this ticket to the appropriate team.

**Comment by ala-yna** [comment]
@joshkimux - Going to go ahead and close this! We switched to the action link, and we'll be changing the icon to green in an in-progress PR.
